lib/fastlane/setup/setup_ios.rb

```diff
@@ -72,6 +79,29 @@ def generate_appfile
       Helper.log.info "Created new file '#{path}'. Edit it to manage your preferred app metadata information.".green
     end
 
+    def default_run_produce
+      Helper.log.info "Running produce..."
```

Comment:
This will always run, right? This must be super scary for users, they run `fastlane` the first time and it's already **modifying** and **creating** things on the Developer Portal. 

Do we already have the summary table which the user can confirm? We should be very transparent of what `fastlane` will do next before it actually does it.